exec/exec.go

```diff
@@ -126,11 +126,12 @@ func runExec(command string) (string, error) {
 // randomized effect by sleeping for the random interval.
 func randomizeSleep(i int) {
 	rand.Seed(time.Now().UnixNano())
-	r := minRandomSleep + rand.Intn(i-minRandomSleep+1) // ensures that the random range is [1, i]. To avoid sleeping for 0 seconds.
+	ms := i * 1000
+	r := minRandomSleep + rand.Intn(ms-minRandomSleep+1) // ensures that the random range is [1, i]. To avoid sleeping for 0 seconds (0 ms).
```

Comment:
Should the +1 be +1000 here?

Reply:
Yep we could or we can have it be meaningful like 100 ms. This just ensures that if `i` is 1 (1 second), then `r` ends up being `1000` + `1` -> `1001`. We can have it 1100ms, so it's meaningful. But also ensures that when `i != 1`, there is enough variability, so we also don't want to add a whole second on the original input.